helpick-service/src/main/java/com/infoshare/util/ValidatorEnum.java

```diff
@@ -4,7 +4,7 @@
     ALPHA("[a-zA-Z]+", "Dozwolone tylko litery"),
     POLISHSIGNS("[a-zA-ZżźćńółęąśŻŹĆĄŚĘŁÓŃ -]+", "Dozwolone tylko litery"),
     PHONENUMBER("^\\d{9}$", "Nieprawidłowy numer telefonu, dozwolone 9 cyfr"),
-    EMAIL("^[A-Za-z0-9+_.-]+@[A-Za-z0-9.-]+$", "Poprawny format to: ktos@gdzies.com"),
+    EMAIL("^[A-Za-z0-9+_.-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,9}$", "Poprawny format to: ktos@gdzies.com"),
```

Comment:
usuń backslashe z regex'a, bo  z nimi nie chce działać :)

Reply:
dasz mi proszę znać w którym miejscu nie waliduje poprawnie? jak robię bez backslash'a to mi nie działa, jak mam z backslash'mi to działa